package/Dockerfile

```diff
@@ -15,7 +15,7 @@ ARG ARCH=amd64
 ARG IMAGE_REPO=rancher
 ARG SYSTEM_CHART_DEFAULT_BRANCH=dev
 # kontainer-driver-metadata branch to be set for specific branch other than dev/master, logic at rancher/rancher/pkg/settings/setting.go  
-ARG RANCHER_METADATA_BRANCH=
+ARG RANCHER_METADATA_BRANCH=dev
```

Comment:
This was changed in https://github.com/rancher/rancher/commit/5ced4dc03384f44db3682bf0652d57274b90e02f